plugins/forms/plugin.js

```diff
@@ -277,3 +277,24 @@ CKEDITOR.plugins.add( 'forms', {
 		}
 	}
 } );
+
+/**
+ * Namespace containing helper functions for Forms Plugin.
+ *
+ * @since 4.10.2
+ * @private
+ * @singleton
+ * @class CKEDITOR.plugins.forms
+ */
+CKEDITOR.plugins.forms = {
+	/**
+	* Sets dialog required value.
+	*
+	* @since 4.10.2
+	* @private
+	* @param {CKEDITOR.dom.element} element An element which required attribute is checked.
+	*/
+	_setupFormDialogRequired: function( element ) {
```

Comment:
That name rubs me up the wrong way… Maybe something like `setupRequiredAttribute`?